### PR TITLE
Fix rubocop issues

### PR DESCRIPTION
**Pull request by vfonic**
This PR is created separately from the `project-setup` because it includes many
subjective decisions that I'd like us to first check and agree on.
I don't think enforcing all these rubocop rules is necessary, but I'd like to
have _some_ rubocop rules enforced. It helps avoid discussions/issues/comments
regarding code style and rubocop also sometimes detects potential bugs.

This PR is dependent on #7

### CODE REVIEW

File: app/services/shopify/products/imports/create.rb [hunk 1]
```diff
@@ -2,8 +2,13 @@ module Shopify
   module Products
     module Imports
       class Create
+        attr_reader :shopify_product
+
         def initialize(shopify_data_feed)
           @shopify_data_feed = shopify_data_feed
+          @shopify_product = ShopifyAPI::Product.new(
```
Comment (Argonus) [inline app/services/shopify/products/imports/create.rb]: @vfonic We should have it as lazy.

File: spec/models/shopify/data_feed_spec.rb [hunk 2]
```diff
@@ -2,8 +2,16 @@
 
 RSpec.describe Shopify::DataFeed, type: :model do
   describe 'database' do
-    it { is_expected.to have_db_index([:shopify_object_id, :shopify_object_type]).unique }
```
Comment (Argonus) [inline spec/models/shopify/data_feed_spec.rb]: @vfonic Shoulda specs are usually one line specs.

File: spec/services/shopify/products/data_parsers/base_data_spec.rb [hunk 3]
```diff
@@ -5,7 +5,9 @@
     subject { described_class.prepare_data(shopify_product) }
 
     context 'with sample product' do
-      let!(:shipping_category) { create(:shipping_category, name: 'ShopifyImported') }
+      let!(:shipping_category) do
```
Comment (Argonus) [inline spec/services/shopify/products/data_parsers/base_data_spec.rb]: @vfonic It should be one line.

File: spec/services/shopify/products/imports/create_spec.rb [hunk 4]
```diff
@@ -1,25 +1,35 @@
 require 'spec_helper'
 
 RSpec.describe Shopify::Products::Imports::Create, type: :service do
-  let!(:client) { ShopifyAPI::Base.site = "https://foo:baz@test_shop.myshopify.com/admin" }
+  let!(:client) do
```
Comment (Argonus) [inline spec/services/shopify/products/imports/create_spec.rb]: @vfonic Why this is change to do end block?

File: .rubocop.yml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+   - 'bin/*'
+   - 'db/schema.rb'
+   - 'vendor/bundle/**/*'
+   - 'spec/dummy/**/*'
+  Include:
+    - 'lib/tasks/**/*'
+    - '**/Gemfile'
+    - '**/Rakefile'
+    - '**/Guardfile'
+  TargetRubyVersion: 2.3
```
Comment (Argonus) [inline .rubocop.yml]: @vfonic We'd like to support two ruby versions as in travis.yml file 
rvm:
  - 2.3.1
  - 2.2.5
Could we do that?

Comment (vfonic) [inline .rubocop.yml]: @Argonus of course. Good catch!

I just noticed that gemspec specifies ruby >= 2.2.2. Which one should be the lowest supported ruby vesion?

https://github.com/spark-solutions/spree_shopify_importer/blob/master/spree_shopify_importer.gemspec#L13

Comment (Argonus) [inline .rubocop.yml]: @vfonic  Yeah, you are absolutely right, lets  make>= 2.2.2

Comment (vfonic) [inline .rubocop.yml]: @Argonus I've addressed this comment.

**Pull request merged**